pkgs/development/python-modules/cadquery/default.nix

```
@@ -0,0 +1,88 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyparsing
+, opencascade
+, stdenv
+, python
+, cmake
+, swig
+, ninja
+, smesh
+, freetype
+, libGL
+, libGLU
+, libX11
+, six
+}:
+
+let 
+  pythonocc-core-cadquery = stdenv.mkDerivation rec {
```

No, that was leftover from a copy-paste, thanks for the spot. 

Now lets see, can I incorporate my own suggestions into this batch of changes?
```suggestion
  pythonocc-core-cadquery = stdenv.mkDerivation {
```